lib/tool_shed/galaxy_install/tools/data_manager.py

```diff
@@ -14,16 +14,27 @@ class DataManagerHandler( object ):
     def __init__( self, app ):
         self.app = app
 
+    @property
+    def data_managers_path( self ):
+        tree, error_message = xml_util.parse_xml( self.app.config.shed_data_manager_config_file )
+        if tree:
+            root = tree.getroot()
+            return root.get( 'tool_path', None )
```

Comment:
You're right, didn't think of that, but I surely agree with the title of your last commit, explicit is indeed better than implicit!